manifests/integrations/disk.pp

```diff
@@ -32,7 +32,7 @@
 #      excluded_disk_re     => '/dev/sd[e-z]*'
 #  }
 class datadog_agent::integrations::disk (
-  $use_mount              = 'no',
+  $use_mount              = false,
```

Comment:
This is exactly what should've been from day one, and I we'll make sure some of this stuff is fixed in the upcoming major release. Unfortunately this would break manifests for current users. I don't think this was currently broken... if it is, please open an issue and I'll tackle it ASAP in the bugfix release.